app/jobs/geocode_location_job.rb

```diff
@@ -47,6 +51,6 @@ def enqueue_next_job
   end
 
   def next_location
-    Location.where('latitude IS NULL OR longitude IS NULL').first
+    Location.where(geocode_response: nil).first
```

Comment:
My concern here is that not only yet-to-be-geocoded `Locations`, but also failed-to-be-geocoded `Locations` would result in a `nil` response, and thus be selected over and over for geocoding. 

In the most pathological case you'd have one un-geocoded `Location`, with a typo or otherwise un-geocodable `content`, being attempted for geocoding every 200 ms.

Does that ring true to you? If so I wonder if maybe a simple `geocoded_at` timestamp would be preferable to `geocode_response`. (That would also eliminate the need for the custom Error above.)

But, for better and for worse, that would eliminate the sort of "automatic retry" behavior you'd be getting with this PR's implementation.